### PR TITLE
[New Rule] Suspicious Explorer Child Process

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/429

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/execution_via_explorer_suspicious_child_parent_args.toml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2020/10/29"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/29"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a suspicious Windows explorer child process. Explorer.exe can be abused to launch malicious script or executables from a trusted parent process.
```
Comment (bm11100) [inline rules/windows/execution_via_explorer_suspicious_child_parent_args.toml:10]:
```suggestion
Identifies a suspicious Windows explorer child process. Explorer.exe can be abused to launch malicious scripts or executables from a trusted parent process.
```

Comment (bm11100) [reply rules/windows/execution_via_explorer_suspicious_child_parent_args.toml:10]: may need to lint

**Review by bm11100 — APPROVED** [review]

File: rules/windows/execution_via_explorer_suspicious_child_parent_args.toml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2020/10/29"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/29"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a suspicious Windows explorer child process. Explorer.exe can be abused to launch malicious scripts or executables from a trusted parent process.
```
Comment (brokensound77) [inline rules/windows/execution_via_explorer_suspicious_child_parent_args.toml:10]: lint

File: rules/windows/execution_via_explorer_suspicious_child_parent_args.toml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Execution"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+ process.name in ("cscript.exe", "wscript.exe", "powershell.exe", "rundll32.exe", "cmd.exe", "mshta.exe", "regsvr32.exe") and
+ /* Explorer started via DCOM */
+  process.parent.name == "explorer.exe" and process.parent.args == "-Embedding"
+
```
Comment (brokensound77) [inline rules/windows/execution_via_explorer_suspicious_child_parent_args.toml:28]:
```suggestion
process where event.type in ("start", "process_started") and
  process.name : ("cscript.exe", "wscript.exe", "powershell.exe", "rundll32.exe", "cmd.exe", "mshta.exe", "regsvr32.exe") and
  /* Explorer started via DCOM */
  process.parent.name : "explorer.exe" and process.parent.args : "-Embedding"
```

**Review by brokensound77 — APPROVED** [review]
2 comments then LGTM 👍

**Pull request merged**